openapi-contracts/src/main/com/sinch/sdk/domains/verification/models/v1/status/StatusCall.java

```diff
@@ -0,0 +1,33 @@
+package com.sinch.sdk.domains.verification.models.v1.status;
+
+import com.sinch.sdk.core.utils.EnumDynamic;
+import com.sinch.sdk.core.utils.EnumSupportDynamic;
+import java.util.Arrays;
+import java.util.stream.Stream;
+
+/** The type of the call result. */
+public class StatusCall extends EnumDynamic<String, StatusCall> {
```

Comment:
Regarding the name of the class, I understand that `Status` is a reference to the "Verification Status" and `Call` is a reference to the result of the call. Wouldn't it be more meaningful to rename it `StatusCallResult` or simply `CallResult` ?